### ISSUE TITLE
Loading new sstables by sstables_loader have not been finished.

### ISSUE THREAD

**Issue by aleksbykov** [scylladb/scylla#10093]
*Installation details*
Kernel version: `5.11.0-1028-aws`
Scylla version (or git commit hash): `5.1.dev-0.20220209.5099b1e27 with build-id b0986550af32b8da96b50442a53423047ed91696`
Cluster size: 6 nodes (i3.4xlarge)
Scylla running with shards number (live nodes):
   longevity-tls-50gb-3d-master-db-node-903b3085-1 (34.244.95.162 | 10.0.0.183): 14 shards
   longevity-tls-50gb-3d-master-db-node-903b3085-23 (34.251.114.30 | 10.0.0.201): 14 shards
   longevity-tls-50gb-3d-master-db-node-903b3085-25 (34.246.222.132 | 10.0.3.121): 14 shards
   longevity-tls-50gb-3d-master-db-node-903b3085-28 (34.243.34.92 | 10.0.2.226): 14 shards
   longevity-tls-50gb-3d-master-db-node-903b3085-29 (34.255.11.185 | 10.0.3.217): 14 shards
   longevity-tls-50gb-3d-master-db-node-903b3085-31 (34.241.140.89 | 10.0.2.78): 14 shards
Scylla running with shards number (terminated nodes):
   longevity-tls-50gb-3d-master-db-node-903b3085-4 (3.251.63.157 | 10.0.2.115): 14 shards
   >> List of nodes is larger than 10. See sct log for a full list of nodes.

OS (RHEL/CentOS/Ubuntu/AWS AMI): `ami-0f894d9df1a4e76fc` (aws: eu-west-1)

Test: `longevity-50gb-3days`
Test name: `longevity_test.LongevityTest.test_custom_time`
Test config file(s):
- [longevity-50GB-3days-authorization-and-tls-ssl.yaml]([https://github.com/scylladb/scylla-cluster-tests/blob/2c3e4506cee8b75c0f4e89b0aefb9f335c52d6dd/test-cases/longevity/longevity-50GB-3days-authorization-and-tls-ssl.yaml](https://www.google.com/url?q=https://github.com/scylladb/scylla-cluster-tests/blob/2c3e4506cee8b75c0f4e89b0aefb9f335c52d6dd/test-cases/longevity/longevity-50GB-3days-authorization-and-tls-ssl.yaml&source=gmail-html&ust=1645162078354000&usg=AOvVaw3EaOAmySBOs-wk5omo5lhG))

**Issue description**

====================================

Nemesis Load and Stream,
-  download sstables from s3 storage,`https://s3.amazonaws.com/scylla-qa-team/refresh_nemesis/load_and_stream/node1.keyspace1.standard1.tar.gz`
- current data removed: `sudo rm -f /var/lib/scylla/data/keyspace1/`
- upload downloadeds sstables to upload/ 
- removed schema and manifest files:
   `sudo rm -f /var/lib/scylla/data/keyspace1/standard1-fe4f35a08af411eca1a013abcfe3ec14/upload/schema.cql`
   `sudo rm -f /var/lib/scylla/data/keyspace1/standard1-fe4f35a08af411eca1a013abcfe3ec14/upload/manifest.json`
- Using the api request loading and stream
```
Running command "curl -X POST --header "Content-Type: application/json" --header "Accept: application/json" "http://127.0.0.1:10000/storage_service/sstables/keyspace1?cf=standard1&load_and_stream=true""...
< t:2022-02-12 06:21:02,692 f:base.py         l:222  c:RemoteCmdRunner      p:DEBUG >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
< t:2022-02-12 06:21:02,692 f:base.py         l:222  c:RemoteCmdRunner      p:DEBUG >                                  Dload  Upload   Total   Spent    Left  Speed
```

Load and stream started on node18:
```
2022-02-12T06:21:02+00:00 longevity-tls-50gb-3d-master-db-node-903b3085-18 !    INFO |  [shard 2] sstables_loader - Loading new SSTables for keyspace=keyspace1, table=standard1, load_and_stream=true, primary_replica_only=false
2022-02-12T06:21:02+00:00 longevity-tls-50gb-3d-master-db-node-903b3085-18 !    INFO |  [shard 12] sstables_loader - load_and_stream: started ops_uuid=aae0cbc9-f631-47a1-8912-bfb5f351c945, process [0-1] out of 1 sstables={/var/lib/scylla/data/keyspace1/standard1-fe4f35a08af411eca1a013abcfe3ec14/upload/md-12-big-Data.db}
2022-02-12T06:21:02+00:00 longevity-tls-50gb-3d-master-db-node-903b3085-18 !    INFO |  [shard 9] sstables_loader - load_and_stream: started ops_uuid=70b9ca6a-c5a5-4813-b7fc-3582679f3fc5, process [0-1] out of 1 sstables={/var/lib/scylla/data/keyspace1/standard1-fe4f35a08af411eca1a013abcfe3ec14/upload/md-9-big-Data.db}
```

but it was not finished or message was not appeared in log that load an stream is done.

[node18.log.tar.gz](https://github.com/scylladb/scylla/files/8086076/node18.log.tar.gz)

====================================

Restore Monitor Stack command: `$ hydra investigate show-monitor 903b3085-d4df-424a-b2be-21f1f1cb5b30`
Restore monitor on AWS instance using [Jenkins job]([https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=903b3085-d4df-424a-b2be-21f1f1cb5b30](https://www.google.com/url?q=https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id%3D903b3085-d4df-424a-b2be-21f1f1cb5b30&source=gmail-html&ust=1645162078355000&usg=AOvVaw00vALEN38724RUydAfRmBE))
Show all stored logs command: `$ hydra investigate show-logs 903b3085-d4df-424a-b2be-21f1f1cb5b30`

Test id: `903b3085-d4df-424a-b2be-21f1f1cb5b30`

Logs:
grafana - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_160635/grafana-screenshot-longevity-50gb-3days-scylla-per-server-metrics-nemesis-20220214_160635-longevity-tls-50gb-3d-master-monitor-node-903b3085-1.png](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_160635/grafana-screenshot-longevity-50gb-3days-scylla-per-server-metrics-nemesis-20220214_160635-longevity-tls-50gb-3d-master-monitor-node-903b3085-1.png)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_160635/grafana-screenshot-longevity-50gb-3days-scylla-per-server-metrics-nemesis-20220214_160635-longevity-tls-50gb-3d-master-monitor-node-903b3085-1.png%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_160635/grafana-screenshot-longevity-50gb-3days-scylla-per-server-metrics-nemesis-20220214_160635-longevity-tls-50gb-3d-master-monitor-node-903b3085-1.png)&source=gmail-html&ust=1645162078355000&usg=AOvVaw1kODNzy9VPsgXsDApY22kU)
critical - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/critical-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/critical-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/critical-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/critical-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw2NquG464U8LNnmjQksoGAz)
db-cluster - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/db-cluster-903b3085.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/db-cluster-903b3085.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/db-cluster-903b3085.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/db-cluster-903b3085.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw2gtS_vP7W2xl62hJbQjRwF)
debug - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/debug-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/debug-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/debug-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/debug-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw1ZtAdBWr1YHDw5_Po5sEX6)
email_data - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/email_data-903b3085.json.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/email_data-903b3085.json.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/email_data-903b3085.json.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/email_data-903b3085.json.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw278_6uI0olp8j43Bl6FejQ)
error - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/error-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/error-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/error-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/error-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw0HtDNn9E7VISX44-bwii6l)
event - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/events-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/events-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/events-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/events-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw1HSF9VcOVUTNWirXc0vNwr)
left_processes - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/left_processes-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/left_processes-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/left_processes-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/left_processes-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw2xrL8yDy0YArpzYxgts-x-)
loader-set - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/loader-set-903b3085.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/loader-set-903b3085.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/loader-set-903b3085.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/loader-set-903b3085.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw1K7E1oinhfO2w6JLkWCZtU)
monitor-set - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/monitor-set-903b3085.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/monitor-set-903b3085.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/monitor-set-903b3085.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/monitor-set-903b3085.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw3by9djjU-Qb4WTH13kYxhJ)
normal - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/normal-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/normal-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/normal-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/normal-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw2ngV8ssp5-2_Zbk7fU-9Eg)
output - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/output-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/output-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/output-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/output-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw1CTiatEHtlwptWeLWFutmc)
event - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/raw_events-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/raw_events-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/raw_events-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/raw_events-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw2RNQ5luYJ2cvBlWXJeCWxB)
sct - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/sct-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/sct-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/sct-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/sct-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw3xCyRpoLNYocrdNEeBVNie)
summary - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/summary-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/summary-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/summary-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/summary-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw1IIPNVNTDMkpicJVPFDmvP)
warning - [[https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/warning-903b3085.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/warning-903b3085.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/warning-903b3085.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/903b3085-d4df-424a-b2be-21f1f1cb5b30/20220214_161957/warning-903b3085.log.tar.gz)&source=gmail-html&ust=1645162078355000&usg=AOvVaw3Qsn0K5p-o9E9gHlPfW3Th)

[Jenkins job URL]([https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-50gb-3days/123/](https://www.google.com/url?q=https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-50gb-3days/123/&source=gmail-html&ust=1645162078356000&usg=AOvVaw1RmiwXQxKZk58DVGx6u0Ym))

**Comment by slivne** [comment]
seems like https://github.com/scylladb/scylla/issues/9902

**Comment by asias** [comment]
It is possible the log messages are just missing due to https://github.com/scylladb/seastar/issues/1012

**Comment by slivne** [comment]
@juliayakovlev / @roydahan can we validate this is actually fixed ^^ / working

**Comment by juliayakovlev** [comment]
> @juliayakovlev / @roydahan can we validate this is actually fixed ^^ / working

I run test where LoadAndStream nemesis was performed 32 times. Every run succeeded,
So the issue fixed

**Comment by avikivity** [comment]
Fixed by 45bdb57b05e48fb4e69cd18e11fd3c8fe15528a8
